XamlStyler.Console has outdated dependency on NuGet.CommandLine

**Describe the bug**
The project `XamlStyler.Console` has an outdated dependency on NuGet.CommandLine.

**To Reproduce**

**Expected behavior**

**Screenshots**

**External Configuration**

**Version Info (please complete the following information):**

**Additional context**
Message form output window:
> XamlStyler\src\XamlStyler.Console\XamlStyler.Console.csproj : warning NU1904: Package 'NuGet.CommandLine' 6.8.0 has a known critical severity vulnerability, https://github.com/advisories/GHSA-68w7-72jg-6qpp

--- a/src/XamlStyler.Console/XamlStyler.Console.csproj
+++ b/src/XamlStyler.Console/XamlStyler.Console.csproj
@@ -36,7 +36,7 @@
       <Version>13.0.3</Version>
     </PackageReference>
     <PackageReference Include="NuGet.CommandLine">
-      <Version>6.8.0</Version>
+      <Version>6.12.2</Version>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>

--- a/src/XamlStyler.UnitTests/XamlStyler.UnitTest.csproj
+++ b/src/XamlStyler.UnitTests/XamlStyler.UnitTest.csproj
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
     <RootNamespace>Xavalon.XamlStyler.UnitTests</RootNamespace>
     <Copyright>Copyright © Xavalon 2024</Copyright>
     <Company>Xavalon</Company>
@@ -28,16 +28,16 @@
 <ItemGroup>
     <PackageReference Include="CompareNETObjects" Version="4.83.0">
     </PackageReference>
-    <PackageReference Include="Irony" Version="1.2.0">
+    <PackageReference Include="Irony" Version="1.5.3">
     </PackageReference>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.8.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.12.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3">
     </PackageReference>
-    <PackageReference Include="NUnit" Version="4.0.1">
+    <PackageReference Include="NUnit" Version="4.3.2">
     </PackageReference>
-    <PackageReference Include="NUnit.ConsoleRunner" Version="3.16.3">
+    <PackageReference Include="NUnit.ConsoleRunner" Version="3.19.1">
     </PackageReference>
-    <PackageReference Include="NUnit3TestAdapter" Version="4.5.0">
+    <PackageReference Include="NUnit3TestAdapter" Version="4.6.0">
     </PackageReference>
   </ItemGroup>
 <ItemGroup>

